pkgs/applications/kde/ksanecore.nix

```diff
@@ -0,0 +1,15 @@
+{
+  mkDerivation, lib,
+  extra-cmake-modules, qtbase,
+  ki18n, sane-backends
+}:
+
+mkDerivation {
+  pname = "ksanecore";
+  meta = with lib; {
+    license = licenses.gpl2;
+    maintainers = with maintainers; [ andrevmatos ];
+  };
+  nativeBuildInputs = [ extra-cmake-modules ];
+  buildInputs = [ qtbase ki18n sane-backends ];
+}
```

Comment:
We should do a treewide PR where we properly format the kde files and move meta last.

Reply:
That's what I win by copy-pasting other derivations 🤦
Sorry about it. Amending.

Reply:
Done :heavy_check_mark:

Reply:
I more meant to universally fix this across all packages but not now. We can do this later.